src/js/components/DataTable/doc.js

```diff
@@ -23,7 +23,7 @@ parts.forEach(part => {
 });
 
 const backgroundShape = {};
-parts.forEach(part => {
+[...parts, 'pinned'].forEach(part => {
```

Comment:
There is no description of what it means to 'pin' a single column. I'm not sure I understand what it means.
Edit: I just read one of your previous review comments and I understand that pin for a column is for pinning a column for horizontal scrolling. I think we should mention that.

Reply:
added documentation